.github/PULL_REQUEST_TEMPLATE/simple_pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+<!-- This template should only be used in rare situations. If it is possible that the details will at all effect the ability of other contributors, the detailed pull request template should be used. If your reviewer believes more info is necessary, they may have you complete the detialed_pull_request_template. -->
+
+## Describe your changes
+
+## Issue ticket number and link
+
+## Checklist before requesting a review
+
+- [ ] I have performed a self-review of my code
+- [ ] If it is a core feature, I have added thorough tests.
+- [ ] Do we need to implement analytics?
+- [ ] Will this be part of a product update? If yes, please write one phrase about this update.
```

Comment:
All of the updates that are code won't be available without a product update. We can remove this
```suggestion
```

Reply:
ok, done. If u remember, I wouldn't understand a bit of clarification on this, ty.